components/ui/inputs.tsx

```diff
@@ -413,3 +413,89 @@ export const TextArea: FC<TextAreaProps> = observer(
     },
   ),
 );
+
+export type RangeInputProps = {
+  min?: string | number;
+  max?: string | number;
+  minLabel?: string;
+  maxLabel?: string;
+  step?: string;
+  value?: string;
+  valueSuffix?: string;
+  className?: string;
+  name?: string;
+  onValueChange?: (val: string) => void;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+};
+
+const RangeInput = React.forwardRef(
```

Comment:
Do you think it would be better to put this in a separate file? I'm finding in a bit hard to track down input components because I can't use the file search for them

Reply:
ok